persist/sqlite/consensus_refactored_test.go

```diff
@@ -270,6 +275,21 @@ func (n *testChain) assertContractRevisions(t *testing.T, fcID types.FileContrac
 	}
 }
 
+func (n *testChain) assertEvents(t *testing.T, addr types.Address, expected ...explorer.Event) {
+	t.Helper()
+
+	events, err := n.db.AddressEvents(addr, 0, math.MaxInt64)
+	if err != nil {
+		t.Fatal(err)
+	}
+	testutil.Equal(t, "len(events)", len(expected), len(events))
+
+	for i := range expected {
+		expected[i].Relevant = []types.Address{addr}
```

Comment:
is this necessary? It seems like we often already pass this in correctly in the `explorer.Event`